<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>897</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Ритмичные поставки</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>V0</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="v0Spin">
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>p</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="2">
         <widget class="QPlainTextEdit" name="pEdit">
          <property name="minimumSize">
           <size>
            <width>385</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>minV</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDoubleSpinBox" name="minVSpin">
          <property name="maximum">
           <double>10000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>maxV</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QDoubleSpinBox" name="maxVSpin">
          <property name="maximum">
           <double>100000000.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="2">
         <widget class="QPushButton" name="solveBtn">
          <property name="text">
           <string>Решить итерационным процессом</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2" rowspan="2">
         <widget class="QTableWidget" name="resultTable"/>
        </item>
        <item row="5" column="0" colspan="2">
         <widget class="QPushButton" name="solveDirectBtn">
          <property name="text">
           <string>Решить прямым методом</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>Готово</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>ЗКПР</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>N</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="nSpin">
            <property name="maximum">
             <number>10000000</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>M</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="mSpin">
            <property name="maximum">
             <number>100000000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="jobsTable">
          <column>
           <property name="text">
            <string>dur</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>rel</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="capTable"/>
        </item>
        <item>
         <widget class="QTableWidget" name="demandsTable"/>
        </item>
        <item>
         <widget class="QTableWidget" name="predsTable">
          <column>
           <property name="text">
            <string>preds</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="scheduleTable"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="buildZkprBtn">
            <property name="text">
             <string>buildZkprBtn</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="solveZkprBtn">
            <property name="text">
             <string>solveZkprBtn</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>897</width>
     <height>37</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Расчётный модуль</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
